src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1208,7 +1210,7 @@ public final void testDeletePatronAccountByIdItemByItemIdHoldByHoldIdWithErrors(
       );
   }
 
-  static Stream<Arguments> itemRequestsParams() {
+    static Stream<Arguments> itemRequestsParams() {
```

Comment:
Why has the indentation of this method signature changed?

Reply:
It's a broken formatting, will fix it. Thanks for noticing.